web-ui/src/components/celebrations/Birthdays.jsx

```diff
@@ -1,6 +1,10 @@
 import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
 
+import { Card, CardHeader, Grid, Typography } from '@mui/material';
```

Comment:
The only changes in this file were to reorder/combine some imports. I thought I was going to need other changes in this file, but it turned out none were required.